test/tracing.test.ts

```diff
@@ -154,6 +157,8 @@ describe('tracing', () => {
       maxAttrLength,
       logInjectionEnabled
     );
+    stopTracing();
```

Comment:
What was the reason `stopTracing` was added to tests? Here and at loginjection

Reply:
Well, so I made a mistake while coding, and I mistakenly thought that the tests were not passing because another test was overwriting the globals. However, I left `stopTracing` in place because deregistering is necessary for some tests in `signalfx/otel-js-web`, so I'm expecting that sooner or later it will be needed in this library too.